tools/ExamplesGenerator.ps1

```diff
@@ -0,0 +1,268 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+Param(
+    $ModulesToGenerate = @(),
+    [string] $ModuleMappingConfigPath = (Join-Path $PSScriptRoot "..\config\ModulesMapping.jsonc")
+)
+function Get-GraphMapping {
+    $graphMapping = @{}
+    $graphMapping.Add("v1.0", "examples\v1.0")
+    $graphMapping.Add("beta", "examples\v1.0-beta")
+    return $graphMapping
+}
+
+function Start-Generator {
+    Param(
+        $ModulesToGenerate = @()
+    )
+
+    $GraphMapping = Get-GraphMapping 
```

Comment:
Consider using a hashtable instead of `Get-GraphMapping` function since `$GraphMapping` is only ever used in `Start-Generator`.
```suggestion
    $GraphMapping = @{
        "v1.0" = "examples\v1.0"
        "beta" = "examples\v1.0-beta"
    }
```